index.bs

```diff
@@ -446,6 +451,30 @@ A {{DatagramDuplexStream}} object has the following internal slots.
 :: The getter steps are:
      1. Return [=this=].`[[Writable]]`.
 
+: <dfn for="DatagramDuplexStream" attribute>incomingMaxAge</dfn>
+:: The getter steps are:
+     1. Return [=this=]'s [=[[IncomingDatagramsExpirationDuration]]=].
+:: The setter steps are:
+     1. Set [=this=]'s [=[[IncomingDatagramsExpirationDuration]]=] to the given value.
```

Comment:
Should we check that the value is > 0 and not-NaN here?

Reply:
Done. According to https://heycam.github.io/webidl/#idl-double and https://heycam.github.io/webidl/#idl-unrestricted-double we don't need to care about inf and NaN.

Reply:
Maybe we want to explicitly allow Infinity as a MaxAge?